atlas/packages/components/src/components/VideoPreview/VideoPreview.tsx

```diff
@@ -1,47 +1,86 @@
-import React from "react"
+import React from "react";
 
-import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles"
-import Avatar from "./../Avatar"
+import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles";
+import Avatar from "./../Avatar";
 
 type VideoPreviewProps = {
-  title: string
-  channel?: string
-  channelImg?: string
-  showChannel?: boolean
-  poster?: string
-  onClick?: any
-  onChannelClick?: any
-} & VideoPreviewStyleProps
+	title: string;
+	channel?: string;
+	channelImg?: string;
+	showChannel?: boolean;
+	showMeta?: boolean;
+	time?: string;
```

Comment:
Do we want this to be a string and give the responsibility of time formatting to the consumer? I would expect this to be a timestamp/`Date`/`luxon`/something else that holds the date.

Reply:
Yes absolutely. This is just placeholder content for now, so I wanted to use the most basic things possible to get a feel of how things look like.

Reply:
Okay, thanks for clearing that up, we may change the type later once we decide on what to use to format dates.

One more suggestion I have would be changing the field name to something more specific. `time` can be ambiguous, something like `createdAt` could be a standard prop name for every field like this. If you feel that this should be done outside of this PR then I'm fine with that as well.